src/Config.php

```diff
@@ -179,7 +179,7 @@ class Config
      *
      * @var array<string, string>
      */
-    private static $executablePaths = [];
+    private static $executablePaths = ["php" => PHP_BINARY];
```

Comment:
I'd prefer an exception for PHP to be made directly in the getExecutablePath() method so that when (inevitably) code moves around in there, this change doesn't cause issues.